developer-guides/quick-start/README.md

```diff
@@ -19,27 +20,19 @@ You may notice build WARNINGs related to _peer dependencies_  or other transitiv
 2. Install meteor
 
     `curl https://install.meteor.com/ | sh`
-
     There is no need to install `node` or `npm`, as meteor already includes them.  Verify by:
-
     `meteor node -v`
-
     `meteor npm -v`
 
 3. Get rocket.chat code
-
-    `git clone https://github.com/RocketChat/Rocket.Chat.git`
+- `git clone https://github.com/RocketChat/Rocket.Chat.git` (if you wish to clone your own fork, replace with your repo link which may look like __*https://github.com/your-user-name/Rocket.Chat.git*__).
+- `cd Rocket.Chat` (or as per the name of your cloned repository).
 
 4. Install modules
-
-    `cd Rocket.Chat`
-
     `meteor npm install`
 
 5. Start building (can take 5 minutes, please be patient)
-
-    `meteor npm start`
-
+- `meteor npm start` (Warning: It may take a while to build so please be patient).
```

Comment:
not necessary at all, please revert

already mentioned in heading 5

Reply:
true